egs2/TEMPLATE/enh_diar1/enh_diar.sh

```diff
@@ -737,7 +737,7 @@ if ! "${skip_eval}"; then
                         ${_ref_scp} \
                         ${_inf_scp} \
                         --ref_channel ${ref_channel} \
-                        --flexible_numspk True
```

Comment:
Just a note, I changed it because bollean args cannot be passed in this way. If some value is passed, the variable would be parsed as True, even here "False" is intended to be used.